wear/src/main/AndroidManifest.xml

```diff
@@ -122,6 +123,19 @@
 
             <meta-data android:name="androidx.wear.tiles.PREVIEW"
                 android:resource="@drawable/favorite_entities_tile_example" />
+
+            <meta-data
+                android:name="com.google.android.clockwork.tiles.MULTI_INSTANCES_SUPPORTED"
+                android:value="true" />
+
+            <meta-data
+                android:name="com.google.android.clockwork.tiles.PROVIDER_CONFIG_ACTION"
+                android:value="_" />
```

Comment:
This will crash the system UI when I tap 'Edit' for the tile on my Samsung Galaxy Watch 4... Could we provide an empty activity that immediately finishes instead to prevent this (or even deeplink to the relevant settings screen)?

```
2023-07-28 16:06:32.125  8163-8163  AndroidRuntime          com.samsung.android.wearable.sysui   E  FATAL EXCEPTION: main
                                                                                                    Process: com.samsung.android.wearable.sysui, PID: 8163
                                                                                                    android.content.ActivityNotFoundException: No Activity found to handle Intent { act=_ cat=[android.intent.category.DEFAULT,com.google.android.clockwork.tiles.category.PROVIDER_CONFIG] pkg=io.homeassistant.companion.android.debug (has extras) }
                                                                                                    	at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:2071)
                                                                                                    	at android.app.Instrumentation.execStartActivity(Instrumentation.java:1727)
                                                                                                    	at android.app.Activity.startActivityForResult(Activity.java:5329)
                                                                                                    	at androidx.activity.ComponentActivity.startActivityForResult(ComponentActivity.java:574)
                                                                                                    	at android.app.Activity.startActivityForResult(Activity.java:5287)
                                                                                                    	at androidx.activity.ComponentActivity.startActivityForResult(ComponentActivity.java:560)
                                                                                                    	at android.app.Activity.startActivity(Activity.java:5673)
                                                                                                    	at android.app.Activity.startActivity(Activity.java:5626)
                                                                                                    	at com.google.android.clockwork.sysui.common.content.SafeBackgroundActivityStarter.startActivity(SafeBackgroundActivityStarter.java:29)
                                                                                                    	at com.google.android.clockwork.sysui.mainui.module.dashboard.tile.edit.DashboardTileEditPresenter.lambda$launchSettings$1(DashboardTileEditPresenter.java:116)
                                                                                                    	at com.google.android.clockwork.sysui.mainui.module.dashboard.tile.edit.DashboardTileEditPresenter.lambda$launchSettings$1$DashboardTileEditPresenter(Unknown Source:0)
                                                                                                    	at com.google.android.clockwork.sysui.mainui.module.dashboard.tile.edit.-$$Lambda$DashboardTileEditPresenter$PmODxYROBl1w4Y5b5682SBVq1Lk.onLoaded(Unknown Source:4)
                                                                                                    	at com.google.android.clockwork.sysui.mainui.module.dashboard.tile.DashboardEditModeTilesLoader$1.onSuccess(DashboardEditModeTilesLoader.java:53)
                                                                                                    	at com.google.android.clockwork.sysui.mainui.module.dashboard.tile.DashboardEditModeTilesLoader$1.onSuccess(DashboardEditModeTilesLoader.java:50)
                                                                                                    	at com.google.android.clockwork.common.concurrent.AbstractCwFutureListener.run(AbstractCwFutureListener.java:34)
                                                                                                    	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:462)
                                                                                                    	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
                                                                                                    	at com.google.android.clockwork.common.concurrent.InstrumentedTask.run(InstrumentedTask.java:72)
                                                                                                    	at android.os.Handler.handleCallback(Handler.java:938)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:99)
                                                                                                    	at android.os.Looper.loop(Looper.java:246)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:7690)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:593)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:995)
```

Reply:
Ohh, I didn't test this, thanks for noticing! I'll try implementing the deep link, 'cause that's the proper way 😄

Reply:
I gave this a shot and came up with something. It can be found on this branch: https://github.com/marazmarci/home-assistant-android/tree/config-shortcut-tile-action

Unfortunately, I couldn't find an "Edit" function for Tiles on my Wear OS 3.5 TicWatch Pro 5 anywhere, so I couldn't really test it.

I have an old Fossil Gen 5 with Wear OS 2, on which I saw a gear icon under the Tile, and I could do a minimal test, where the HA debug app was in a logged-out state. I discovered an NPE and fixed it in 16efe653, but that's all. To log in to HA, I'd have to go through the hassle of factory resetting the watch and pairing it with another phone, which I totally don't have time for currently.

Could you please test the changes from the branch I linked above on your Samsung Galaxy Watch 4? 🙏

Reply:
Thanks, I'll check this tomorrow.

Reply:
Works great! Just need to actually use the tile ID. I modified the code for the activity to use the following which seems to be working as expected:

```kotlin
override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    
    val tileId = intent.extras?.getInt("com.google.android.clockwork.EXTRA_PROVIDER_CONFIG_TILE_ID")
    if (tileId == null || tileId == 0) {
        finish()
    }

    val intent = HomeActivity.getShortcutsTileSettingsIntent(
        context = this,
        tileId = tileId!!
    )
    startActivity(intent)
    finish()
}
```

Reply:
Nice, thanks! I'll update it soon!

Reply:
You requested a review but this wasn't added yet.

Reply:
Ohh sorry, I messed up my git stuff 🤦

Fixed in 49547cab and f230de02